mkdocs.yml

```diff
@@ -150,7 +150,7 @@ nav:
     - Docs for Iter8 v0.7: https://iter8-tools.github.io/iter8/0.7/
 - User guide:
   - Topics:
```

Comment:
Should include "Docs for Iter8 v0.9".

Reply:
Not needed, since it will be part of the drop down on top. The others will not be.

Reply:
Where is this drop down? I pulled the PR and ran the docs locally. I am not sure where to look for it.

Reply:
The dropdown will show up in `iter8.tools`, but will not show up locally (the is a quirk of how `material mkdocs` works).